### ISSUE TITLE
Failed OSC update

### ISSUE THREAD

**Issue by bkornpob** [astrocatalogs/astrocats#93]
Hi,

I was trying to search for some SNe (e.g., SN 2015bn, SN 2017egm, and SN 2018bsz) on the website, and it did not find the object. The picture is attached here.

Actually, I found out this issue a couple of days ago, and I generated a post somewhere that I think it was not here. In case you see the earlier post, I am reporting the same issue.

![osc](https://user-images.githubusercontent.com/13034318/50712397-5a91cf00-103f-11e9-898b-f645cc0dea07.png)

**Comment by guillochon** [comment]
Yes you can see that only 20k SNe are in the database (there are usually 50k). I will investigate.

**Comment by guillochon** [comment]
This is I believe fixed now, though I'm going to keep the issue open until we have a regular update that completes successfully. Thanks for bringing this to our attention!
